usagov/vote-gov-nvrf-app
# Cypress Set Up

Purpose: Setting up Cypress into the nvrf repo 

testing:
1. pull down branch 
2. cd into `testing`
3. run `npm install` 
4. run `npm cy:open` and select the `helloWorld.cy.js` file and check that cypress is running and test is passing with the sign in command
